kolena/_experimental/dataset/__init__.py

```diff
@@ -13,8 +13,59 @@
 # limitations under the License.
 from kolena._experimental.dataset._dataset import fetch_dataset
 from kolena._experimental.dataset._dataset import register_dataset
+from kolena.workflow.annotation import BitmapMask
+from kolena.workflow.annotation import BoundingBox
+from kolena.workflow.annotation import Polygon
+from kolena.workflow.annotation import ScoredBoundingBox
+from kolena.workflow.annotation import ScoredBoundingBox3D
+from kolena.workflow.annotation import ScoredClassificationLabel
+from kolena.workflow.annotation import ScoredLabeledBoundingBox
+from kolena.workflow.annotation import ScoredLabeledBoundingBox3D
+from kolena.workflow.annotation import ScoredLabeledPolygon
+from kolena.workflow.annotation import ScoredPolygon
+from kolena.workflow.annotation import SegmentationMask
+from kolena.workflow.asset import Asset
+from kolena.workflow.asset import BinaryAsset
+from kolena.workflow.asset import ImageAsset
+from kolena.workflow.asset import PlainTextAsset
+from kolena.workflow.asset import PointCloudAsset
+from kolena.workflow.ground_truth import GroundTruth
```

Comment:
GroundTruth, Inference, and Model should not be included. Classes in `plot` is also not relevant to `dataset`.

Reply:
If we want to offer a new path to import all of these, what do you propose? I'm open to pausing this part of the PR (keeping to just rename TEST_SAMPLE to DATAPOINT, and handling the other imports in a follow up. Should these also be relocated to another module or do we want `kolena.workflow.plot` to still be used?

 @yifanwu-kolena @dylangrandmont  What are your thoughts here?

Reply:
just renaming TEST_SAMPLE to DATAPOINT sounds good to me.

i will not worry the importing paths until dataset is in good stable shape

Reply:
done